frontend/packages/ceph-storage-plugin/src/components/modals/volume-snapshot-modal/volume-snapshot.ts

```diff
@@ -12,3 +12,7 @@ export const snapshotTypes = {
 };
 
 export const cronLink = 'https://en.wikipedia.org/wiki/Cron#Overview';
+
+export const days = Array.from(Array(31), (e, i) => i + 1);
```

Comment:
@a2batic what if I choose 31st day under Monthly option ? Wont it fail for few months ?
How is this being handled ?

Reply:
Please use date objects for all date related operations. 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date

Reply:
@yuvalgalanti what do you suggest to handle this?

Reply:
I will suggest selecting the date for today by default

Reply:
@a2batic so, first of all, it looks great, I would say that if a user selects the 31st on the "monthly" option he practically selects the "max" selection for each month, so in months where 31st does not exist, we will select the 30th. 

one other thing, why are we showing the horizontal scroll?